code/components/gta-net-five/src/MumbleVoice.cpp

```diff
@@ -700,30 +700,119 @@ static HookFunction hookFunction([]()
 
 			if (id >= 0 && id < 31)
 			{
-				vtConfigs[id] = {};
-				g_mumbleClient->UpdateVoiceTarget(id, vtConfigs[id]);
+				if (g_mumble.connected)
+				{
+					vtConfigs[id] = {};
+					g_mumbleClient->UpdateVoiceTarget(id, vtConfigs[id]);
+				}
 			}
 		});
 
-		fx::ScriptEngine::RegisterNativeHandler("MUMBLE_CLEAR_VOICE_TARGET_CHANNELS", [](fx::ScriptContext& context)
+        fx::ScriptEngine::RegisterNativeHandler("MUMBLE_REMOVE_VOICE_TARGET_CHANNEL", [](fx::ScriptContext& context)
```

Comment:
How did this end up indented with spaces, ignoring the .editorconfig?

Reply:
I actually have no idea why, I'll try and fix it when I force push